### DISCUSSION TITLE
Is there a way to update workflow templates via git ?

### DISCUSSION THREAD

**Discussion by artemus717** [argoproj/argo-workflows#13147] [Q&A] [answered]
I store templates in a git repository. Is there a way to update workflow templates in ArgoWorkflows via git ?

**Comment by Joibel** [comment, marked as answer]
Use ArgoCD to synchronise them with your cluster? That is what we do.

**Reply by artemus717** [reply]
You mean sync template as Kubernetes resource ?

**Reply by artemus717** [reply]
Oh it works for me. Usually people can not find this way to manage template via ArgoCD.
Many thanks.

**Reply by agilgur5** [reply]
This option is also mentioned in the [`WorkflowTemplate` docs](https://argo-workflows.readthedocs.io/en/latest/workflow-templates/#gitops-via-argo-cd)
